### ISSUE TITLE
Feature request: Supertask sequential execution of tasks

### ISSUE THREAD

**Issue by rickvanderzwet** [hashtopolis/server#821]
When running a supertask it would be nice to have sequential execution, which can save computing time. For example a supertask consist of for tasks (with priorities). 1) <most-common> (40) 2) wordlist + rules (30) 3) brute-force. (20)  4) brute-force (20)

I would like to be able to execute them in order to ensure the most likely candidates are tried (and exhausted) before the heavy lifting takes place. This could for example be done by a flag within the supertask telling the scheduler to hold-off tasks if execution of higher priorities tasks are not yet finished.

**Comment by zyronix** [comment]
Currently it is possible to set the priority in the SuperTask of the pretast which are part of it. Agents will then first work on the pretask with the highest priority. But lets say that one of those pretask have a high prio and isfor example a small task; a new agent comes along and starts working on the SuperTask; it will start on the next pretask in line.

What you are asking for is to prevent this and make sure the first task is completed before any other task is started?

**Comment by rickvanderzwet** [comment]
> What you are asking for is to prevent this and make sure the first task is completed before any other task is started?
Yes indeed, having a flag per-supertask optionto force this behavior if needed would be nice.

**Comment by ObsidianOracle** [comment]
Its really an edge case. As a workaround make sure you just assign one agent to the supertask. This would trigger the subtasks to be handled sequentially.
